resources/js/Pages/SignUp.vue

```diff
@@ -3,6 +3,11 @@
     <Menu />
     <Head title="Sign up" />
     <section>
+        <b-message
+                    type="is-danger"
+                    v-if="form.errors.email">
+                        {{ form.errors.email }}
+        </b-message>
```

Comment:
```suggestion
        <b-message type="is-danger" v-show="form.hasErrors">
            <ul>
                <li v-for="msg in form.errors">
                    {{ msg }}
                </li>
            </ul>
        </b-message>
```
Your only checked if the email field had an error, not if the other had one.